src/main/drivers/sdio_f4xx.c

```diff
@@ -1574,16 +1574,20 @@ void SD_Initialize_LL(DMA_Stream_TypeDef *dma)
     RCC->AHB1ENR |= RCC_AHB1ENR_GPIOCEN | RCC_AHB1ENR_GPIODEN;
 
     const IO_t d0 = IOGetByTag(IO_TAG(PC8));
+#ifndef USE_SDIO_1BIT
```

Comment:
Agreed with @jflyper, this should be done as runtime configuration, to make it work for universal targets.